tests/ignite/contrib/metrics/test_cohen_kappa.py

```diff
@@ -0,0 +1,208 @@
+import os
+
+import numpy as np
+import pytest
+import torch
+from sklearn.metrics import cohen_kappa_score
+
+import ignite.distributed as idist
+from ignite.contrib.metrics import CohenKappa
+from ignite.engine import Engine
+from ignite.exceptions import NotComputableError
+
+
+def test_no_update():
+    ck = CohenKappa()
+
+    with pytest.raises(NotComputableError):
+        ck.compute()
+
+
+def test_input_types():
+    ck = CohenKappa()
+
+    with pytest.raises(ValueError):
+        ck.update((torch.randint(0, 2, size=(10, 2, 3)).long(), torch.arange(0, 10).float()))
+
+    with pytest.raises(ValueError):
+        ck.update((torch.rand(10,).float(), torch.randint(0, 2, size=(10, 3, 1)).long(),))
+
+    with pytest.raises(ValueError):
+        ck.update((torch.randint(0, 4, size=(10, 2, 3)).long(), torch.randint(0, 2, size=(10, 5)).long()))
+
+
+def test_check_shape():
+    ck = CohenKappa()
+
+    with pytest.raises(ValueError):
+        ck._check_shape((torch.randint(0, 2, size=(10, 1, 5, 12)).long(), torch.randint(0, 2, size=(10, 5, 6)).long()))
+
+    with pytest.raises(ValueError):
+        ck._check_shape((torch.randint(0, 2, size=(10, 1, 6)).long(), torch.randint(0, 2, size=(10, 5, 6)).long()))
+
+    with pytest.raises(ValueError):
+        ck._check_shape((torch.randint(0, 2, size=(10, 1)).long(), torch.randint(0, 2, size=(10, 5, 2)).long()))
+
+
+@pytest.mark.parametrize("weights", [None, "linear", "quadratic"])
+def test_cohen_kappa_all_weights(weights):
+    size = 100
+    np_y_pred = np.random.randint(0, 2, size=(size, 1), dtype=np.long)
+    np_y = np.random.randint(0, 2, size=(size, 1), dtype=np.long)
+    np_ck = cohen_kappa_score(np_y, np_y_pred)
+
+    ck_metric = CohenKappa(weights=weights)
+    y_pred = torch.from_numpy(np_y_pred)
+    y = torch.from_numpy(np_y)
+
+    ck_metric.reset()
+    ck_metric.update((y_pred, y))
+    ck = ck_metric.compute()
+
+    assert ck == pytest.approx(np_ck)
+
+
+def test_cohen_kappa_wrong_weights_type():
+    with pytest.raises(ValueError):
+        CohenKappa(weights=7)
+
+    with pytest.raises(ValueError):
+        CohenKappa(weights="dd")
+
+    with pytest.raises(ValueError):
+        CohenKappa(weights="ss")
+
+    with pytest.raises(ValueError):
+        CohenKappa(weights="l")
+
+    with pytest.raises(ValueError):
+        CohenKappa(weights="q")
+
+    with pytest.raises(ValueError):
+        CohenKappa(weights="z")
```

Comment:
I think all those tests can be removed

Reply:
Okay done

Reply:
@KickItLikeShika remove the lines from 75 - 82.